android-components/components/compose/cfr/build.gradle

```diff
@@ -32,6 +32,10 @@ android {
         freeCompilerArgs += "-Xjvm-default=all"
     }
 
+    lint {
+        disable 'MissingTranslation'
+    }
```

Comment:
what is this for?

Reply:
That is for ignoring lint MissingTranslation error for the `mozac_cfr_a11y_title` string added. It applies only for the compose cfr component.

Reply:
meaning that it will ignore the lint warning when a string is used in this component/module, but the string will still be flagged for localization?

Do you know why this lint disable wasn't needed before?

Reply:
> meaning that it will ignore the lint warning when a string is used in this component/module, but the string will still be flagged for localization?

Exactly.

> Do you know why this lint disable wasn't needed before?

From what I have seen, it was already used in [AC here](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/android-lint.gradle), and it is applied to multiple components like this:
```
apply from: '../../../android-lint.gradle'
```
I could do the same here, but I am not sure if we want all the other warnings disabled. Let me know what you think.

Reply:
Let's keep the lint changes light for now